pepper-apis/src/main/java/org/broadinstitute/ddp/model/user/EnrollmentStatusType.java

```diff
@@ -7,18 +7,39 @@
 
 public enum EnrollmentStatusType {
 
+    /**
+     * Means participant is registered with study but not yet enrolled. Typically this is after they have created an
+     * account but before giving consent.
+     */
     REGISTERED(true, false, false, true),
+    /**
+     * Means participant is fully enrolled in study. Typically this is after they have completed consent.
+     */
     ENROLLED(true, false, false,  true),
+    /**
+     * Means participant withdrew from study before being enrolled.
+     */
     EXITED_BEFORE_ENROLLMENT(false, true, true, false),
+    /**
+     * Means participant withdrew from study after being enrolled.
+     */
     EXITED_AFTER_ENROLLMENT(false, true, true, false),
-    CONSENT_SUSPENDED(true, false, false, true);
+    /**
+     * Means the consent participant given is no longer valid, and thus suspended. Participant is no longer considered
+     * enrolled until they re-consent. Typically this can happen when a minor participant ages up. Their parental
+     * consent will be suspended and they will need to give their own consent.
+     */
+    CONSENT_SUSPENDED(true, false, false, true),
+    /**
+     * Means participant has reached the end of participation in the study. This is very similar to `ENROLLED`. They are
+     * considered to be fully enrolled in the study and can continue to view/edit their data, but will not be
+     * participating in future activities (e.g. will not receive any more kits).
+     */
+    COMPLETED(true, false, false, true);
```

Comment:
I think we should set `shouldReceiveCommunications` here to `false`.  If they are completed, they should not receive any comms.

Reply:
Actually, now that I've seen the test for getting the result, now I think maybe we should keep communications as `true`...Let me double check with PIs...

Reply:
Oh and I totally forgot about the fact that they _could_ request and get symptomatic kits.  So let's leave this as `true` unless I hear otherwise...